Fatal error: exception Failure("duplicate function foo")
